specification/machinelearningservices/data-plane/Microsoft.MachineLearningServices/preview/2019-05-01/artifact.json

```diff
@@ -0,0 +1,1799 @@
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "hardDelete",
+            "in": "query",
+            "description": "If set to true. The delete cannot be revert at later time.",
+            "required": false,
+            "type": "boolean",
+            "default": false
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The Artifact metadata deleted successfully."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Artifact": {
+      "description": "Details of an Artifact.",
+      "type": "object",
+      "properties": {
+        "artifactId": {
+          "description": "The identifier of an Artifact. Format of ArtifactId - {Origin}/{Container}/{Path}.",
+          "type": "string"
+        },
+        "origin": {
+          "description": "The origin of the Artifact creation request. Available origins are 'ExperimentRun', 'LocalUpload', 'WebUpload', 'Dataset' and 'Unknown'.",
+          "type": "string"
+        },
+        "container": {
+          "description": "The name of container. Artifacts can be grouped by container.",
+          "type": "string"
+        },
+        "path": {
+          "description": "The path to the Artifact in a container.",
+          "type": "string"
+        },
+        "etag": {
+          "description": "The Etag of the Artifact.",
+          "type": "string"
+        },
+        "createdTime": {
+          "format": "date-time",
+          "description": "The Date and Time at which the Artifact is created. The DateTime is in UTC.",
+          "type": "string"
+        },
+        "dataPath": {
+          "$ref": "#/definitions/DataPath"
+        }
+      }
+    },
+    "DataPath": {
+      "type": "object",
+      "properties": {
+        "dataStoreName": {
+          "type": "string"
+        },
+        "relativePath": {
+          "type": "string"
+        },
+        "sqlDataPath": {
+          "$ref": "#/definitions/SqlDataPath"
+        }
+      }
+    },
+    "SqlDataPath": {
+      "type": "object",
+      "properties": {
+        "sqlTableName": {
+          "type": "string"
+        },
+        "sqlQuery": {
+          "type": "string"
+        },
+        "sqlStoredProcedureName": {
+          "type": "string"
+        },
+        "sqlStoredProcedureParams": {
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/StoredProcedureParameter"
+          }
+        }
+      }
+    },
+    "StoredProcedureParameter": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "value": {
+          "type": "string"
```

Comment:
What are the valid values for a Boolean?

Reply:
can you elaborate your question? I dont see any property of type boolean here.

Reply:
I assume that the [`type`](https://github.com/Azure/azure-rest-api-specs/pull/5734/files/feb6896215b7840ab9a7931f2f705c88d4d0bdb7#diff-8b2ce1459b986799fe42985fc534cda9R1449) attribute indicates what type the value if the `value`  attribute is supposed to be interpreted as? And one of the types for `type` is "Boolean".

Reply:
sure I will look into it.

Reply:
The valid values of boolean are 'true' and 'false'